src/frontend/src/lib/components/hero/HeroContent.svelte

```diff
@@ -66,14 +65,12 @@
 		</div>
 	{/if}
 
-	{#if actions}
```

Comment:
This is not related to this PR per sé, right?

Reply:
right! i put it here just to simplofy, but I can remove it once we merge this one, what do you prefer?

Reply:
All good